src/clipboard-copy-element.js

```diff
@@ -31,7 +31,7 @@ function copyTarget(document: Document, id: string) {
 }
 
 function applyHint(button: Element) {
-  const hint = button.getAttribute('data-copied-hint')
+  const hint = button.getAttribute('hint')
```

Comment:
So `hint` is only ever going to affect `aria-label`, right? Do you think it would ever apply in a broader since? Just trying to brainstorm other names:

* `aria-copied-label`
* `copied-label`

Reply:
I like how `copied-label` matches its `aria-label` pair. Maybe we should avoid using anything with an `aria-` prefix, like `aria-copied-label`, so we don't step on future accessibility attributes.